include/boost/http_proto/impl/serializer.hpp

```diff
@@ -12,9 +12,6 @@
 
 #include <boost/http_proto/detail/except.hpp>
 #include <boost/buffers/range.hpp>
-#include <iterator>
-#include <new>
-#include <utility>
```

Comment:
We should keep `#include <utility>` as we're using `std::forward` directly here.